README.md

```diff
@@ -1,11 +1,13 @@
 # NEMSIO
 
-Performs I/O for the NCEP models using NEMS.
+Byte addressable I/O C package (originally written by Mark Iredell and Jun Wang)
```

Comment:
The code and API for reading binary files is in Fortran.
The description here is for BACIO.